.travis.yml

```diff
@@ -1,8 +1,20 @@
 language: scala
 scala:
-  - "2.10.6"
```

Comment:
Is Scala 2.10 supported?

Reply:
Do we still want to support Scala 2.10 ? Some of the dependencies already remove support from Scala 2.10.

Reply:
No, let's drop.

Reply:
2.10.0 is from 2013-01-04 and 2.10.5 is from 2015-03-05.

If somebody is looking for 2.10 support, they can use the original sPDF.